frontend/src/pages/Home/index.tsx

```diff
@@ -1,70 +1,110 @@
-            <h1 className="text-white text-6xl mb-0">ExplainaBoard</h1>
-            <p className="text-white text-2xl ">
-              Understand and debug the state-of-the-art in NLP.
-            </p>
-            <div className="relative flex flex-col min-w-0 break-words bg-white py-3 mx-20 rounded-lg">
-              <h2 className="text-3xl">Do you want to?</h2>
-              <ul className="text-xl text-left mx-16">
-                <li className="mb-1">
-                  <b>Upload and analyze</b> your own system results on an
-                  existing dataset:
-                  <br />
-                  Use the{" "}
-                  <a href="https://github.com/neulab/explainaboard_client">
-                    ExplainaBoard client
-                  </a>{" "}
-                  or go to Systems and click New
-                </li>
-                <li className="mb-1">
-                  View a list of <b>supported datasets</b>:
-                  <br />
-                  Go to Datasets
-                </li>
-
-                <li className="mb-1">
-                  <b>View systems</b> and compare/contrast their strengths and
-                  weaknesses:
-                  <br />
-                  Go to Systems, select a system or systems and click Analysis
-                </li>
-                <li className="mb-1">
-                  <b>Ask a question</b>, report an issue, or contribute to
-                  ExplainaBoard:
-                  <br />
-                  File an issue on the github for the{" "}
-                  <a href="https://github.com/neulab/explainaboard">
-                    ExplainaBoard backend
-                  </a>
-                  {" or "}
-                  <a href="https://github.com/neulab/explainaboard_web">
-                    web UI
-                  </a>
-                  .
-                </li>
-              </ul>
+      <div className="w-full h-full bg-white">
+        <div
+          className="w-full h-50 bg-center bg-cover mb-4"
+          style={{ backgroundImage: `url(${logo})` }}
+        >
+          <div className="bg-mask h-full flex items-center">
+            <div>
+              <div className="w-full text-center ml-auto mr-auto mt-3">
+                <h1 className="text-6xl header-style">ExplainaBoard</h1>
+              </div>
+              <div className="w-80 text-center ml-auto mr-auto mb-3">
+                <p className="text-2xl text-font text-bold text-light">
+                  Want to understand why your machine learning systems are
+                  working or failing?
+                </p>
+                <p className="text-2xl text-font text-bold text-light">
+                  ExplainaBoard can help!
+                </p>
+              </div>
             </div>
           </div>
         </div>
+        <div className="items-center mb-4">
+          <Row className="flex">
+            <Col offset={1} span={6} className="items-center flex">
```

Comment:
Could you please extract the cards to a component?

Reply:
That's a good idea. Thanks